public/static/docs/glossary.js

```diff
@@ -25,14 +20,14 @@ Initialized by running \`dvc init\` in the **workspace** (typically in a Git
 repository). It will contain the
 [\`.dvc/\` directory](/doc/user-guide/dvc-files-and-directories) and
 [DVC-files](/doc/user-guide/dvc-file-format) created with commands such as
-\`dvc add\` or \`dvc run\`. It may also be a Git repository.
+\`dvc add\` or \`dvc run\`.
       `
     },
     {
       name: 'DVC Repository',
       match: ['DVC repository', 'DVC repositories'],
       desc: `
-**DVC project** initialized using \`dvc init\` in a Git repository. It will
+**DVC project** initialized (with \`dvc init\`) in a Git repository. It will
```

Comment:
@shcheklein maybe we should merge the terms "DVC project" and "DVC repository" (into just project)? They're basically the same, but one of them specifies that it's in a Git repo.

Reply:
yep, good idea. And we can mention that it might or might not be Git controlled.

Reply:
or even mention there as well that some commands, options etc expect it to be Git as well as DVC and usually it's obvious from the context

Reply:
OK! Terms merged in c55a2fe7 (updated all docs accordingly).

> we can mention that it might or might not be Git controlled

This is already mentioned in the definition of _DVC project_.

> some commands, options etc expect it to be Git as well as DVC and usually it's obvious from the context

Added this note into the definition. It's a little long now though:

![image](https://user-images.githubusercontent.com/1477535/73684419-c2bf4400-4689-11ea-8a8b-6157892dad03.png)

Reply:
Should I remove the last sentence? "Some DVC commands..." I don't feel like it belongs in the short definition of the term. It will be mentioned in the appropriate cmd refs anyway.

Reply:
Yeah. Removed in 7ef09685 (part of #966)

Reply:
Discussion revived in https://github.com/iterative/dvc.org/pull/931#pullrequestreview-354892810.